config/config.toml

```diff
@@ -187,7 +199,7 @@ key_store.mapping_key = "RelevantOracleMappingAddress"
 [opentelemetry]
 
 # Timeout in seconds for the OpenTelemetry exporter
```

Comment:
Should probably change this comment from "seconds" now that it can accept arbitrary units.